consensus/hotstuff/validator/validator_test.go

```diff
@@ -314,13 +355,27 @@ func (vs *VoteSuite) TestVoteSignatureError() {
 	assert.False(vs.T(), model.IsInvalidVoteError(err), "internal exception should not be interpreted as invalid vote")
 }
 
+func (vs *VoteSuite) TestVoteInvalidSignerID() {
```

Comment:
can we add a minimal godoc?

Reply:
added documentations for test functions https://github.com/onflow/flow-go/pull/1830/commits/3a5ee936f6d86c451c674dd66d6e33f5671cdd42